en/docs/install-and-setup/installation-guide/basic-health-checks.md

```diff
@@ -34,3 +38,30 @@ cURL command:
 ``` java
     <ns:getVersionResponse xmlns:ns="http://version.services.core.carbon.wso2.org"><return>WSO2 API Manager-2.6.0</return></ns:getVersionResponse>
 ```
+## APIM Analytics Profiles
+
+Basic health checks can be performed on APIM analytics profiles by connecting to relevant ports. See the following table for the ports that can be used for health checks.
+
+| APIM Analytics Profile | Ports that can be used for health checks |
+|------------------------|------------------------------------------|
+| Analytics Worker       | 9091 (HTTP), 9444 (HTTPS)                |
+| Analytics Dashboard    | 9092 (HTTP), 9643 (HTTPS)                |
+
+APIM Analytics by default is shipped with the simple service named `health`. This service returns the status as a json string as `{"status":"healthy"}`.
+
+The cURL command format, a sample cURL command and the response from the `health` service are given below.
```

Comment:
Shall we mention that the service is accessible in worker profile in 9091 and dashboard in 9092 port.

Reply:
Resolved with cf4fb6cef2491e8abf9b04bcefce56e1da975735